validator/src/main/java/com/google/daq/mqtt/registrar/Registrar.java

```diff
@@ -655,6 +655,28 @@ private void loadSchema(String key) {
     }
   }
 
+  private void loadSiteDefaults() {
+    this.siteDefaults = null;
+
+    if (!schemas.containsKey(METADATA_JSON))
+      return;
+
+    File siteDefaultsFile = new File(siteDir, SITE_DEFAULTS_JSON);
+    try (InputStream targetStream = new FileInputStream(siteDefaultsFile)) {
+      schemas.get(METADATA_JSON).validate(OBJECT_MAPPER.readTree(targetStream));
+    } catch (ProcessingException | ValidationException e) {
```

Comment:
I'm typically lazy with my catch -- you can just catch (Exception e) and then have one line since there's no functional difference between them (just adds context and then walks it up the stack)

Reply:
Ok but I got this from LocalDevice line 294. Should we clean that up too?

Reply:
There's a slight difference -- the code in LocalDevice.java is different in the two cases, where here it's the same in all cases. Doesn't really matter here (it's functionally correct) -- just something that jumped out at me.